libkbfs/block_retrieval_worker_test.go

```diff
@@ -247,4 +248,5 @@ func TestBlockRetrievalWorkerShutdown(t *testing.T) {
 	require.True(t, shutdown)
 	w.Shutdown()
 	require.True(t, shutdown)
+	cancel()
```

Comment:
If you keep this, why not a `defer cancel()` right where this is created, so if any early returns are added in between we still cancel it?